README.md

```diff
@@ -31,6 +34,13 @@ For Save Track(s):
 - Holding no modifying keys will prompt to save the current track
 - Holding **Shift** will prompt to save all tracks (including hidden)
 
+For Disc Switching:
+- If you change the number of discs in the internal changer, you must restart the program for it to take effect
```

Comment:
Why? Do you plan to fix this?

Reply:
Because you're changing the hardware internals that's running. I'll consider changing that in the future.